dojo/tools/snyk/parser.py

```diff
@@ -92,25 +105,30 @@ def get_item(vulnerability, test):
         severity = vulnerability['severity'].title()
 
     if 'id' in vulnerability:
-        references = "Custom SNYK ID: {}\n\n".format(vulnerability['id'])
+        references = "<b>Custom SNYK ID</b>: https://app.snyk.io/vuln/{}\n\n".format(vulnerability['id'])
```

Comment:
Avoid adding HTML formatting in `references` attribute. If you want to have some formatting use Markdown syntax
```suggestion
        references = "**Custom SNYK ID**: <https://app.snyk.io/vuln/{}>\n\n".format(vulnerability['id'])
```